### ISSUE TITLE
Ensure sufficient contrast for credential metadata colours

### ISSUE THREAD

**Issue by smncd** [sirosfoundation/wallet-frontend#268]
Credential metadata can provide background and text colour. We use these in the wallet UI in connection with the credentials, such as in the presentation flow. However, we currently don't check that these colours actually have a sufficient amount of contrast, and if not, replace the text colour. There should be a function for this already in the `config/branding.ts` file.
